lib/routes/podwise/collections.ts

```diff
@@ -0,0 +1,46 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+
+export const route: Route = {
+    path: '/explore',
+    categories: ['multimedia'],
+    example: '/podwise/explore',
+    radar: [
```

Comment:
Personal
Checking
Savings & CDs
Credit Cards
Home Loans
Personal Loans
Auto Loans
Premier````

````
Education & Tools
Investing & Wealth Management
Small Business
Commercial Banking
Corporate & Investment Banking
Good afternoon
Sign on to manage your accounts.

Username

Password
 Save username
 Enroll
Forgot username or password?
Security Center
Privacy, Cookies, and Legal

$300 checking bonus on us
New customers open an eligible checking account with qualifying electronic deposits
Get started >>
 
Want a $125 checking bonus?
Open a new Clear Access Banking account and complete the qualifying requirements

Learn more
 
Earn 20,000 bonus points
Plus, earn 3X points on many everyday categories with $0 annual fee. Terms apply.

Learn more
 
Open a savings account
Explore our savings accounts and find the right fit for you

Get started
Interest rates today


Check rates
 
Earn up to 60,000 bonus points
on qualifying purchases. Terms apply.

Learn more
‍
Financial guidance and support

Get tools. Get tips. Get peace of mind.

Discover digital tools to help you budget, save, manage credit, and more
Access the toolkit


Reimagine. Navigate. Move forward.

Discover steps that may help you manage life-changing events
Manage life events


Save. Invest. Retire well.

Discover how to start saving to meet your retirement goals
Get ready for retirement


Banking in the palm of your hand
Our Wells Fargo Mobile® app gives you fast and secure access to your finances
Check your account balance
View your latest FICO® Score1
Send and receive money with Zelle®2